frontend/src/components/activity/CardContentNode.vue

```diff
@@ -15,30 +15,52 @@
           />
         </div>
 
-        <v-toolbar-title v-else>
+        <v-toolbar-title v-if="!editInstanceName">
           {{ instanceOrContentTypeName }}
         </v-toolbar-title>
+        <v-spacer v-if="!editInstanceName" />
 
-        <v-spacer />
-
-        <v-menu v-if="!layoutMode && !disabled" bottom left offset-y>
-          <template #activator="{ on, attrs }">
-            <v-btn icon v-bind="attrs" v-on="on">
-              <v-icon>mdi-dots-vertical</v-icon>
+        <v-tooltip
```

Comment:
Can the tooltip and the icon go into it's own component? I guess we want to use a similar icon/tooltip in many places.

Was earlier looking into using [floating-vue](https://floating-vue.starpad.dev/guide/#quick-start), as vuetify makes it quite complex to show a tooltip. However, if we have the tooltip neatly packed into a component which we can use everywhere, it doesn't really matter.

Reply:
I agree